src/SwedbankPay.Sdk/PaymentInstruments/Swish/SwishPaymentSaleRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SwedbankPay.Sdk.Common;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Swish
+{
+    public class SwishPaymentSaleRequest
+    {
+        public SwishPaymentSaleRequest(Msisdn msisdn)
+        {
+            Transaction = new SwishPaymentSaleTransaction(msisdn);
+        }
+
+
+        public SwishPaymentSaleTransaction Transaction;

```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)